### DISCUSSION TITLE
v5.53.0

### DISCUSSION THREAD

**Discussion by sokra** [webpack/webpack#14265] [General]
# Features

* add `node.__dirname/__filename: "warn-mock"` which warns on usage (will be enabled in webpack 6 by default)

# Bugfixes

* add `stream/web` to Node.js externals
* fix IgnorePluginSchema
* fix builds with persistent caching taking 1 minute to build at least

# Experiments

* add `experiments.futureDefaults` to enable defaults for webpack 6

<hr /><em>This discussion was created from the release <a href='https://github.com/webpack/webpack/releases/tag/v5.53.0'>v5.53.0</a>.</em>

**Comment by KyleAMathews** [comment]
This seems to have changed the validation for the ignore plugin? https://github.com/gatsbyjs/gatsby/issues/33216

**Reply by alexander-akait** [reply]
@KyleAMathews Because previously it will not work, nothing happens, it is fix

**Reply by KyleAMathews** [reply]
so `checkResource` is required?

**Reply by alexander-akait** [reply]
Code is simple https://github.com/webpack/webpack/blob/main/lib/IgnorePlugin.js#L42, you need `checkResource` or `resourceRegExp` (with optional `contextRegExp`)

**Reply by KyleAMathews** [reply]
false alarm, turns out we'd fixed this already on master but it hadn't been released yet https://github.com/gatsbyjs/gatsby/pull/33092

**Reply by RomanHotsiy** [reply]
This is a breaking change in a minor update. Can we revert this or make it a warning instead of crash?

@KyleAMathews @alexander-akait while this is fixed on the latest version fo gatsby all the older versions of gatsby (some people can't upgrade immediately)  are broken for the new installs forever 😩

**Reply by alexander-akait** [reply]
but your plugin just doesn't work, it is bug...

**Reply by RomanHotsiy** [reply]
Ignore plugin not working doesn't break any production code. In fact it was used in gatsby to ignore moment locales which most of the people even don't have in their gatsby sites as a dependency.

So from the user perspective I see it as:

Before:

- plugin didn't work, nothing happend, maybe some extra files getting into the bundle

Now:

- build crashes and users have no idea what to do about it

**Reply by RomanHotsiy** [reply]
@alexander-akait I see you point though.

**Reply by alexander-akait** [reply]
We can fix this, but the problem is that while we make the release, developer still get problems with installation, and until everything is fine it will take time, maybe better to do fast patch for Gatsby, we will try not to change schema in the future in this way, so as not to repeat similar problems

**Reply by RomanHotsiy** [reply]
I am not gatsby developer. I am user and we are stuck on an older version now and can't upgrade.

**Reply by RomanHotsiy** [reply]
@alexander-akait does it take a while to get a new release?

**Reply by alexander-akait** [reply]
yes, most likely only after weekends

**Comment by popuguytheparrot** [comment]
Somewhere i can read what else will be in Webpack 6?
